.github/workflows/build.yml

```diff
@@ -14,18 +14,18 @@ on:
         default: https://github.com/KSPModdingLibs/KSPLibs/raw/main/KSP-1.12.5.zip
       dependency-identifiers:
         type: string
-      msbuild-dependency-target:
-        type: string
       solution-file-path:
         type: string
+      no-msbuild:
+        type: boolean
+        default: false
+      no-ckan:
+        type: boolean
+        default: false
```

Comment:
Why is this preferred over specifying the msbuild target to run?  Many mods actually have no dependencies.  I’d prefer that installing dependencies is still opt-in.

also, considering that you can set the default input value, Boolean switches should be named I the affirmative so that you don’t have to mentally parse double negatives